projects/demo/src/pages/recipes/contenteditable/contenteditable-doc.component.ts

```diff
@@ -0,0 +1,25 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
```

Comment:
i have zero experience in angular, so i just copied TextArea recipe page, and changed <textarea> to <div contenteditable="true">